spec/models/post_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.new(name: 'MySelf', photo: 'someurl.com', bio: 'Do what you must', posts_counter: 0)
+  subject do
+    Post.new(
+      title: 'XRP gets clarity',
+      text: 'XRP is the Standard',
+      author: user
+    )
+  end
+
+  before { subject.save }
+
+  it 'post should contain title' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'post should have an author' do
+    subject.author = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'post cannot have negative likes count' do
+    subject.likes_counter = -4
+    expect(subject).to_not be_valid
+  end
+
+  it 'post cannot have negative comment count' do
+    subject.comments_counter = -4
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Kindly test the method that returns 5 recent comments as per project requirements. We are told to test the model methods and validations.


![methods](https://github.com/abel-tefera/ror-blog-app/assets/53124548/f48bb43d-a633-4af6-8050-ce21e584579a)